iop/dev9/atad/src/ps2atad.c

```diff
@@ -888,29 +894,37 @@ static int ata_device_set_transfer_mode(int device, int type, int mode)
 /* Export 9 */
 /* Note: this can only support DMA modes, due to the commands issued. */
 int ata_device_sector_io(int device, void *buf, u32 lba, u32 nsectors, int dir)
+{
+    return ata_device_sector_io_internal(device, buf, (u64)lba, nsectors, dir);
+}
+
+int ata_device_sector_io_internal(int device, void *buf, u64 lba, u32 nsectors, int dir)
```

Comment:
could we use ata_device_sector_io64 instead of ata_device_sector_io_internal, and add a DECLARE_IMPORT for it in atad.h , in order to let homebrew that use atad able to use 64bits lba ?

Reply:
Good idea. This can be added in this PR, or in a later one when the import is needed.

Reply:
This is good!

Reply:
Changed to ata_device_sector_io64 and exported in the latest changes